src/cdpy/ml.py

```diff
@@ -37,3 +37,52 @@ def describe_all_workspaces(self, env=None):
             if ws_desc is not None:
                 resp.append(ws_desc)
         return resp
+    
+    def list_workspace_access(self, name: str = None, crn: str = None, env: str = None):
+        resp = self.sdk.call(
+            svc='ml', func='list_workspace_access', ret_field='users', ret_error=True,
+            workspaceName=name,
+            environmentName=env,
+            workspaceCrn=crn
+        )
+        if isinstance(resp, CdpError):
```

Comment:
These should be handled with Squash I think, unless it cannot understand these particular errors?

Reply:
I believe I can convert the first "catch" to use Squelch, i.e. `UNKNOWN`, but for the second error, i.e. `INVALID_ARGUMENTS`, I want the error to be thrown -- honor the `ret_error` -- but update the error `messages` with something more descriptive before returning the error itself.